manual/googlechrome-extensions/googlechrome-github-expandinizr/googlechrome-github-expandinizr.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-github-expandinizr</id>
+    <version>1.0.0</version>
```

Comment:
I wasn't planning on matching the version number.  I don't "think" the url's etc change per version, and as a result, better to leave it at 1.0.0.  Thoughts?